tests/console/hwloc_testsuite.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's hwloc tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Run testsuite included in hwloc sources
+# Maintainer: Thomas Blume <tblume@suse.com>
+
+use base "consoletest";
+use warnings;
+use strict;
+use testapi;
+use utils;
+
+sub run() {
+    # install hwloc testsuite
+    become_root();
```

Comment:
I have seen tests which failing because they rely on fact that previous test suppose to select root console then you move them in some different place and they start failing. So nevertheless I think it is nice to have this become_root in anycase. It make test more isolated